lib/consul/async/version.rb

```diff
@@ -1,5 +1,5 @@
 module Consul
   module Async
-    VERSION = '1.35.1'.freeze
+    VERSION = '1.36.1'.freeze
```

Comment:
1.36.0 instead?

Reply:
Done, initially applied chameleon rule, but in fact it was done only once.